src/systems/systems/src/hydraulic/trimmable_horizontal_stabilizer.rs

```diff
@@ -0,0 +1,1066 @@
+                context,
+                min_actuator_angle,
+                total_actuator_range_angle,
+                max_elec_motor_speed,
+                elec_motor_over_trim_actuator_ratio,
+            ),
+            trim_wheel: TrimWheels::new(
+                context,
+                total_actuator_range_angle / total_trim_wheel_range_angle,
+                min_trim_wheel_angle,
+                total_trim_wheel_range_angle,
+            ),
+            ths_assembly: TrimmableHorizontalStabilizerHydraulics::new(
+                context,
+                min_ths_deflection,
+                ths_deflection_range,
+            ),
+        }
+    }
+
+    pub fn update(
+        &mut self,
+        context: &UpdateContext,
+        electric_controller: &impl PitchTrimActuatorController,
+        manual_controller: &impl ManualPitchTrimController,
+        pressures: [Pressure; 2],
+    ) {
+        self.pitch_trim_actuator.update(
+            context,
+            electric_controller,
+            manual_controller,
+            &self.ths_assembly,
+        );
+
+        self.trim_wheel.update(&self.pitch_trim_actuator);
+
+        self.ths_assembly.update(
+            context,
+            pressures,
+            self.pitch_trim_actuator.position_normalized(),
+        )
+    }
+
+    pub fn position_normalized(&self) -> Ratio {
+        self.pitch_trim_actuator.position_normalized()
+    }
+
+    pub fn left_motor(&mut self) -> &mut impl Actuator {
+        self.ths_assembly.left_motor()
+    }
+
+    pub fn right_motor(&mut self) -> &mut impl Actuator {
+        self.ths_assembly.right_motor()
+    }
+}
+impl SimulationElement for TrimmableHorizontalStabilizerAssembly {
+    fn accept<T: SimulationElementVisitor>(&mut self, visitor: &mut T) {
+        self.pitch_trim_actuator.accept(visitor);
+        self.trim_wheel.accept(visitor);
+        self.ths_assembly.accept(visitor);
+    }
+}
+
+struct TrimmableHorizontalStabilizerHydraulics {
+    deflection_id: VariableIdentifier,
+    hydraulic_motors: [HydraulicDriveMotor; 2],
+
+    speed: AngularVelocity,
+    actual_deflection: Angle,
+
+    min_deflection: Angle,
+    max_deflection: Angle,
+    deflection_range: Angle,
+
+    is_at_max_up_spool_valve: bool,
+    is_at_max_down_spool_valve: bool,
+}
+impl TrimmableHorizontalStabilizerHydraulics {
+    const HYDRAULIC_MOTOR_POSITION_ERROR_BREAKPOINT: [f64; 7] =
+        [-50., -0.5, -0.1, 0., 0.1, 0.5, 50.];
+    const HYDRAULIC_MOTOR_SPEED_REGULATION_COEF_MAP: [f64; 7] = [-1., -1., -0.6, 0., 0.6, 1., 1.];
+
+    // Gain to convert hyd motor speed to ths deflection speed
+    const HYD_MOTOR_SPEED_TO_THS_DEFLECTION_SPEED_GAIN: f64 = 0.000085;
+
+    const MAX_DEFLECTION_FOR_FULL_OPEN_SPOOL_VALVE_DEGREES: f64 = 0.4;
+
+    pub fn new(context: &mut InitContext, min_deflection: Angle, deflection_range: Angle) -> Self {
+        Self {
+            deflection_id: context.get_identifier("HYD_FINAL_THS_DEFLECTION".to_owned()),
```

Comment:
This SimVar should be documented.

Reply:
This one is only local to wasm, which value is written to ELEVATOR TRIM POSITION